resources/js/components/global/GlobalLeaders.vue

```diff
@@ -7,14 +7,15 @@
 						<th class="has-text-centered" style="width: 15%;">{{ $t('location.position') }}</th>
 						<th>{{ $t('location.name') }}</th>
 						<th class="has-text-centered">{{ $t('location.xp') }}</th>
-                        <th class="has-text-centered hide-mobile">Socials</th>
+                        <th class="has-text-centered hide-mobile">Social</th>
```

Comment:
This was not a bug :D

Reply:
I didn't do this change, it's coming from your commits on master. lol